src/screens/DiscoveryScreenV2/hooks/useDiscovery.js

```diff
@@ -18,10 +19,13 @@ const useDiscovery = () => {
     }));
   }, [discovery]);
 
-  const exhangeFollower = (newUserLists, willFollow, userId) => {
+  const exchangeFollower = (newUserLists, willFollow, userId) => {
     const indexUser = newUserLists.findIndex((item) =>
       item.user ? item.user.user_id === userId : item.user_id === userId
     );
+
+    if (indexUser < 0) return null;
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function name `exchangeFollower` has been corrected from the previous `exhangeFollower`. This improves readability and understanding of the code. Good job!



<!-- This is an auto-generated comment by OSS CodeRabbit -->